00_introduction/22_architecture_root_server.txt

```diff
@@ -14,19 +14,32 @@ Operating System.
 Any CPU provided on commodity servers available nowaday should be fine. On
 a VM server, you should dedicate at least one core to the Rudder server.
 
-At least 1024 MB of RAM must be available on the server, depending on the
-base requirements of your operating system.
+The RAM needed by the server depends essentially of the number of nodes. 
```

Comment:
The amount of RAM needed by the server depends mainly on the number of managed nodes.